docs/make.jl

```diff
@@ -12,20 +12,15 @@ pages = [
     "Background Information" => [
         "ODE Solvers" => "background/ode_solvers.md",
         "Newtons Method" => "background/newtons_method.md",
-    ],
-    "Algorithm formulations" => [
-        "algo_formulations/lsrk.md",
-        "algo_formulations/ssprk.md",
-        "algo_formulations/ark.md",
-        "algo_formulations/mrrk.md",
```

Comment:
Can we keep the name `algo_formulations`? It's more descriptive than `background`.

Reply:
Sure, done!